operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -58,7 +58,24 @@ const (
 // HTTPSource specifies how to download the CRD tarball via either HTTP or HTTPS protocol.
 type HTTPSource struct {
 	// URL specifies the URL of the CRD tarball resource.
+	// +kubebuilder:validation:Required
```

Comment:
```suggestion
	// +required
```
We usually use `+required` to keep consistent?